src/sentry/options/defaults.py

```diff
@@ -324,6 +324,8 @@
 register("store.load-shed-save-event-projects", type=Any, default=[])
 register("store.load-shed-process-event-projects", type=Any, default=[])
 register("store.load-shed-symbolicate-event-projects", type=Any, default=[])
+register("store.symbolicate-event-dlq-never", type=Any, default=[])
+register("store.symbolicate-event-dlq-always", type=Any, default=[])
```

Comment:
> I went looking at the code of `register` and now wonder why `type=Any`? It seems if you don't provide a type it'll derive it from the `default` arg and end up with `Sequence`?

I'll admit I mostly followed the example of the existing kill switches that use the same implementation and just copied what they did, but I don't see there being any issues with being explicit about the type on these two options. I'll use `Sequence` in this case.

> If we need to stick with options for this we'd have to implement external locking for the set part.

Would you happen to have any ideas on where or how else you'd rather see this implemented? I'm starting to develop the conclusion that options may not be the best place to implement the behaviour we want from these "options" for the LPQ, especially when it comes to the `-selected` list. I want to avoid having to build something from scratch, but I'm unsure about what alternatives we have to options if there are any.

Reply:
From talking to people I'm coming to the conclusion that these two which are actually killswitches should probably stay with this and they will have to swallow the O(n) cost.  For the selected list we can move to redis as storage backend for this.